### DISCUSSION TITLE
multi tenant database

### DISCUSSION THREAD

**Discussion by repzlaw** [spatie/laravel-activitylog#779] [Q&A]
i run a multi tenant database using inertia and laravel and vue
when i include use spatie in my models that are under the namespace App (Models in App directory), it works.
But when i use it in my models under the namespace App/Tenant (Models in App/Tenant directory), it doesnt work.
please why is this?

here is the code

<?php

namespace App\Tenant;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasOne;
use Spatie\Activitylog\Traits\LogsActivity;

class School extends Model
{
    use LogsActivity;
   

    protected $fillable = [
        'sch_name', 'sch_title',
        'sch_email', 'sch_phone',
        'sch_address', 'sch_type',
        'image', 'lga_id ',
        'country',
        'incorp',
    ];

    protected static $logFillable = true;
    }

**Comment by Gummibeer** [comment]
Hey, could you please add the code?

If you use the spatie tenant package @freekmurze @brendt @sebastiandedeyne or any other @spatie developer could help you. 🤔

**Comment by ShahanurSharif** [comment]
I wish to add company_id column, but no luck

**Reply by Gummibeer** [reply]
Answered in #815
